core/controllers/topic_editor.py

```diff
@@ -183,8 +186,17 @@ def get(self, topic_id):
             raise self.PageNotFoundException(
                 Exception('The topic with the given id doesn\'t exist.'))
 
+        skill_ids = copy.deepcopy(topic.uncategorized_skill_ids)
+
+        for subtopic in topic.subtopics:
+            skill_ids.extend(copy.deepcopy(subtopic.skill_ids))
+
+        skill_id_to_description_dict = (
+            skill_services.get_skill_descriptions_by_ids(skill_ids))
+
         self.values.update({
-            'topic': topic.to_dict()
+            'topic_dict': topic.to_dict(),
```

Comment:
Should we file an issue to update the naming elsewhere as well (in all other files)?

Reply:
Thanks! Let's try and do that before this PR goes in, so that it doesn't fall through the cracks.

Reply:
Done, created [issue](https://github.com/oppia/oppia/issues/5176).